### PR TITLE
Add serilog and logger to UserLoginConsumer

### PR DESCRIPTION
**Pull request by djino97**

### CODE REVIEW

File: src/UserService/Startup.cs [hunk 1]
```diff
@@ -184,7 +185,7 @@ public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
 
             app.UseRouting();
 
-            app.UseMiddleware<TokenMiddleware>();
+            //app.UseMiddleware<TokenMiddleware>();
```
Comment (KostinP) [inline src/UserService/Startup.cs:188]: Must not forget to return

Comment (djino97) [reply src/UserService/Startup.cs]: Fixed

File: src/UserService.Broker/Consumers/UserLoginConsumer.cs [hunk 2]
```diff
@@ -36,30 +42,22 @@ private object GetUserCredentials(IUserCredentialsRequest request)
...
+            {
+                _logger.LogError(exception, exception.Message);
 
-                dbUserCredentials = _credentialsRepository.GetUserCredentialsByUserId(dbUser.Id);
-
-                if (dbUserCredentials == null)
-                {
-                    throw new NotFoundException($"User credentials for user with email: '{request.LoginData}' was not found.");
-                }
+                throw new Exception(exception.Message);
```
Comment (KostinP) [inline src/UserService.Broker/Consumers/UserLoginConsumer.cs:60]: `throw new Exception(exception.Message);` - it's bad

**Review by KostinP — CHANGES_REQUESTED** [review]

File: src/UserService.Broker/Consumers/UserLoginConsumer.cs (hunk 2 above)
Comment (KostinP) [reply src/UserService.Broker/Consumers/UserLoginConsumer.cs:60]: When we throw an error, it is written to the logger, so we don't have to do it.

Comment (djino97) [reply src/UserService.Broker/Consumers/UserLoginConsumer.cs]: Reworked

**Review by KostinP — APPROVED** [review]

**Review by TIove — APPROVED** [review]

**Review by 24eeeee — APPROVED** [review]

**Comment by DarkFelas** [comment]
no need any more we will use seq

**Pull request closed without merging**